src/linear_algebra/free_module/pid.lean

```diff
@@ -608,6 +608,10 @@ begin
   simp [hi]
 end
 
+instance (F : Type u) [comm_ring F] [algebra F R] (b : basis ι R S) {I : ideal S} (hI : I ≠ ⊥) (i) :
+  module F (R ⧸ ideal.span ({I.smith_coeffs b hI i} : set R)) :=
+by apply_instance -- quotient.module' _
+
```

Comment:
What was the purpose of adding this instance if `apply_instance` can already find it?

Reply:
See https://github.com/leanprover-community/mathlib/pull/19084#discussion_r1208093970

> It's weird we need to add a shortcut instance that is already found by typeclass inference.